tests/test_williamson.py

```diff
@@ -20,7 +20,7 @@ class TestParseHTML:
     @pytest.mark.parametrize(
         "index, expected",
         [
```

Comment:
Instead of deleting the line with the parameters for the existing test case, could you just add the parameters for your new test case on the next line? The same goes for all the deleted lines below this comment.

Example:
```
    [
        (0, "Name, Realistic Fake"),
        (1, "Nelson, Charlie A"),
    ]
```

Reply:
Good call, added them back in. I pulled in the latest version of the file, so replaced some values that seem to have been updated since the test page was pulled

Reply:
Deleting williamson_0.html effectively deletes the prior tests. That could be justified if we knew that I made an error when collecting williamson_0.html or that no pages will ever get served in the future that look like williamson_0.html. But, do we know that for sure? 

If not, you could add your new test HTML file as williamson_2.html instead of saving it over williamson_0.html. I have been replacing the parties' names with "realistic" fake names in order to protect their privacy--the county court records aren't Google-able, but the master branch of this repo probably is, so I don't think all the personal info from the county database is necessarily good to include here.

Reply:
Totally get the point about anonymizing data, can do. 

I didn't really explain this well, but my reasoning for replacing williamson_0.html was that it seemed to be based on case number 1JC-21-0008. The code as-is successfully returns a .json for that case number when run on the console, but the tests for williamson_0.html fail with  AttributeError: 'NoneType' object has no attribute 'parent'. I believe this is due to the original williamson_0.html file being linted somehow (the indentation is different than the raw html when you read it from the page). The new williamson_0.html file is that same case number (which now has some more information), but with the raw string from the query_case_id method directly written to a file. We can discuss more tonight